src/rules/__tests__/fixtures/tsconfig-withmeta.json

```
@@ -0,0 +1,6 @@
+{
+    "extends": "./tsconfig.json",
+    "compilerOptions": {
+        "emitDecoratorMetadata": true,
+    }
+}
```

```suggestion
}

```

or something 😅 add a trailing newline